concordium-sdk/src/main/java/com/concordium/sdk/responses/transactionstatus/BakerDelegationRemoved.java

```diff
@@ -0,0 +1,34 @@
+package com.concordium.sdk.responses.transactionstatus;
+
+import com.concordium.grpc.v2.BakerEvent;
+import com.concordium.sdk.responses.AccountIndex;
+import com.concordium.sdk.types.AccountAddress;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+
+/**
+ * Removed an existing delegator.
+ */
+@Getter
+@ToString
+@SuperBuilder
+@EqualsAndHashCode(callSuper = true)
+public class BakerDelegationRemoved extends AbstractBakerResult {
+
+    private final AccountIndex delegatorId;
+
+    public static BakerDelegationRemoved from(BakerEvent.DelegationRemoved bakerDelegationRemoved, AccountAddress sender) {
+        return BakerDelegationRemoved
+                .builder()
+                .delegatorId(AccountIndex.from(bakerDelegationRemoved.getDelegatorId()))
+                .account(sender)
+                .build();
```

Comment:
In the original event, there is not `AccountAddress` in the data. I think we should keep to that here as well 🙂

Reply:
For all the already existing events, this address is set to the transaction `sender` address and not taken from the event data. For example, none of `BakerEvent`s have `account` attribute https://github.com/Concordium/concordium-grpc-api/blob/88183a8f5e0a3208f4da65d46e0c56d7ec5f14f7/v2/concordium/types.proto#L1133

Maybe the reason for adding `account` is that this model represents not an event but a transaction result, therefore it makes sense to know who was the sender. I wish we could ask the creators.

Reply:
You're right - let's just continue as is then 😄